cmd/runtimetest/main.go

```diff
@@ -320,6 +322,95 @@ func validateROPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func getIDMappings(idType string) ([]rspec.IDMapping, error) {
+	var mapFile string
+	var idMaps []rspec.IDMapping
+	switch idType {
+	case "uid_map":
+		mapFile = "/proc/1/uid_map"
+	case "gid_map":
+		mapFile = "/proc/1/gid_map"
```

Comment:
Can we look at `/proc/self/*_map` instead?  Then we don't have to think about “is it ok to look at PID 1's maps?”.  I can't think of a case where looking at PID 1 would be wrong, but I know looking at `/proc/self/*_map` will be right ;).